pkg/vm/engine/disttae/types.go

```diff
@@ -47,6 +47,44 @@ const (
 	MO_COLUMNS_ID  = 3
 )
 
+// column's index in catalog table
+const (
+	MO_DATABASE_DAT_ID_IDX   = 0
+	MO_DATABASE_DAT_NAME_IDX = 1
+	MO_TABLES_REL_ID_IDX     = 0
+	MO_TABLES_REL_NAME_IDX   = 1
+)
+
+var (
+	MoDatabaseSchema = []string{
+		"dat_id",
+		"datname",
+		"dat_catalog_name",
+		"dat_createsql",
+		"owner",
+		"creator",
+		"created_time",
+		"account_id",
+	}
+	MoTablesSchema = []string{
```

Comment:
I don't known if these columns will contains all of the tableDef（eg:  ViewDef,  CommentDef, IndexDef,  ExternalTable)

Reply:
yes